src/App.jsx

```diff
@@ -4,15 +4,17 @@ import AddTodoForm from './AddTodoForm'
 import { useState } from 'react'
 
 function App() {
+  const [todoList, setTodoList] = useState([]);
 
-  const [newTodo, setNewTodo] = useState("");
+  function addTodo(newTodo) {
+    setTodoList((previousTodoList) => [...previousTodoList, newTodo]);
```

Comment:
Nice, updateState functions can be passed a callback which will receive the previous state as an argument. The usage of previousTodoList here is best practices. Great Job!